### ISSUE TITLE
Seeming problem with Amazon Female first names

### ISSUE THREAD

**Issue by jacoby** [gizmomathboy/hyperborea_apps#11]
I have only encountered this problem when generating names for Amazon women. There seems to be a data alignment issue that gives us 

- (no result) Theodosia , female amazon
- Pelagia (no result) , female amazon
- (no result) (no result) , female amazon

There's also a name starting with P with a comma inside the name. I haven't noticed these problems in other namesets.

**Comment by gizmomathboy** [comment]
Yeah, looks like there was some weird formatting and such in that file.

I've made some changes and it should be fixed.

Give it a try now

**Comment by gizmomathboy** [comment]
another name request yesterday found a problem with male amazon names with I have fixed. Same table format problems I think.
